core/lib/dal/sqlx-data.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I guess you need to update sqlx-cli (prolly you still have 0.5 installed)

Reply:
(Just in case: this file should be removed, it's a side effect of the old `sqlx`)